cms/src/test/java/uk/nhs/digital/apispecs/IsAnyTrueHelperTest.java

```diff
@@ -0,0 +1,155 @@
+
+    @Mock private Options.Buffer buffer;
+    private String item = "some-item";
+
+    @Before
+    public void setUp() throws Exception {
+        initMocks(this);
+    }
+
+    @Test
+    public void rendersBlockWhenCollectionHasAtLeastOneTrueItem() throws IOException {
+
+        Boolean[] params = {false,false,true,false,false};
+        StubOptions options = new StubOptions(params);
+
+        // when
+        final Object buffer = isAnyTrueHelper.apply(item, options);
+
+        // then
+        assertThat("Returns Options.Buffer",
+            buffer,
+            instanceOf(Options.Buffer.class)
+        );
+
+        final Options.Buffer actualBuffer = (Options.Buffer) buffer;
+
+        then(actualBuffer).should().append(TEMPLATE_CONTENT_FROM_THE_POSITIVE_BLOCK);
+    }
+
+    @Test
+    public void rendersBlockWhenCollectionHasNoTrueItem() throws IOException {
+
+        Boolean[] params = {false,false,false,false,false};
+        StubOptions options = new StubOptions(params);
+
+        // when
+        final Object buffer = isAnyTrueHelper.apply(item, options);
+
+        // then
+        assertThat("Returns Options.Buffer",
+            buffer,
+            instanceOf(Options.Buffer.class)
+        );
+
+        final Options.Buffer actualBuffer = (Options.Buffer) buffer;
+
+        then(actualBuffer).should().append(TEMPLATE_CONTENT_FROM_THE_INVERSE_BLOCK);
+    }
+
+    @Test
+    public void throwExceptionWhenCollectionIsEmpty() throws IOException {
+
+        Boolean[] params = {};
+        StubOptions options = new StubOptions(params);
+
+        try {
+            // when
+            isAnyTrueHelper.apply(item, options);
+
+        } catch (Exception e) {
+
+            // then
+            assertThat("Exception in execution",
+                e.getMessage(),
+                containsString("One or more input parameters")
+            );
+        }
+    }
+
+    @Test
+    public void throwExceptionWhenCollectionIsNull() throws IOException {
+
+        Boolean[] params = null;
+        StubOptions options = new StubOptions(params);
+
+        try {
+            // when
+            isAnyTrueHelper.apply(item, options);
+
+        } catch (Exception e) {
+
+            // then
+            assertThat("Exception in execution",
+                e.getMessage(),
+                containsString("One or more input parameters")
+            );
+        }
+    }
+
+    @Test
+    public void throwExceptionWhenCollectionIsNotBoolean() throws IOException {
+
+        Object[] params = {false,"test", 4};
+        StubOptions options = new StubOptions(params);
+
+        try {
+            // when
+            isAnyTrueHelper.apply(item, options);
+
+        } catch (Exception e) {
+
+            // then
+            assertThat("Exception in execution",
+                e.getMessage(),
+                containsString("Required parameter type is Boolean")
+            );
+        }
+    }
+
```

Comment:
Since we are verifying each validation with dedicated test case, then I think that it makes sense to have one for `Validate.noNullElements`.

Reply:
Fixed